tests/manage/mcg/test_write_to_bucket.py

```diff
@@ -69,9 +69,69 @@ def test_write_file_to_bucket(self, mcg_obj, awscli_pod, bucket_factory):
     @pytest.mark.polarion_id("OCS-1949")
     @tier1
     @acceptance
```

Comment:
I think this should not be part of the acceptance. @nimrod-becker @RazTamir what do you think?

Reply:
added the acceptance tag since the tag was applied to the original data reduction test

Reply:
I also suggest removing this from the acceptance suite; Since the test was originally written, it was discovered that there are inconsistencies in data reduction, and it mostly caused false failures (reduction worked, just not in the expected amount)